tests/shared/src/test/scala/caseapp/CaseAppTests.scala

```diff
@@ -8,12 +8,12 @@ import shapeless.{Inl, Inr}
 import utest._
 import caseapp.core.util.Formatter
 
-object Tests extends TestSuite {
+object CaseAppTests extends TestSuite {
```

Comment:
Renamed as this conflicted with https://github.com/lihaoyi/utest/blob/master/utest/src/utest/Tests.scala .